test/FunctionalTests/Microsoft.OData.Core.Tests/ODataNotificationReaderTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataNotificationReaderTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.OData.Tests
+{
+    public class ODataNotificationReaderTests
+    {
+        private MemoryStream stream;
+        private TextReader reader;
+        private IODataStreamListener streamListener;
+
+        public ODataNotificationReaderTests()
+        {
+            this.stream = new MemoryStream();

```

Comment:
This stream and the reader should both really be disposed after each test. I'm not familiar enough with XUnit to know how to do that. Is there a way to have cleanup code called after each test is run?

Reply:
@corranrogue9 To quote from another source... `MemoryStream` does not have any unmanaged resources to dispose, so you don’t technically have to dispose of it. The effect of not disposing a `MemoryStream` is roughly the same thing as dropping a reference to a `byte[]` -- the GC will clean both up the same way.
Specifically for tests, I wouldn't sweat too much over that